gui/src/components/s3/bucket-creation.vue

```diff
@@ -377,6 +377,7 @@ export default class CortxBucketCreation extends Vue {
   private bucketNameTooltipMessage = bucketNameTooltipMessage;
   private bucketUrl = "";
   private noBucketPolicy: boolean;
+  private bucketUrlNone: boolean = false;
```

Comment:
variable name should suggest the type and purpose.

Reply:
Variable names are updated